en/author-guidelines.md

```diff
@@ -14,330 +14,328 @@ skip_validation: true
 
-### Figures
-No matter how short or simple, all lessons benefit from images, particularly screen shots (or partial screen shots) that illustrate what the reader should see as they move through the tutorial. Not only do they make tutorials more "skimmable," figures help show the reader they are doing the right thing. And of course images can save a considerable amount of description in your text.
+Use web-friendly file formats such as .png or .jpg and reduce large images to a maximum of 840px on the longest side. This is important for readers in countries with slower internet speeds.
 
+Images should be saved in a folder with the same name as your lesson .md file. The editor assigned to your lesson can assist you in uploading your images when you submit. 
 
-#### Create a folder
-First, create a folder in which you will store all of your image files  The folder name should be the same as the `LESSON-SLUG` that you have chosen for your lesson file name. The editor assigned to your lesson can assist you in uploading your images to the `ph-submissions` repository when you submit your lesson.
+To insert an image in your text, use the following format:
 
-
-#### Use intelligible filenames
-There are two ways you can name your files. One option is to use consistent, semantically meaningful filenames that clearly indicate what the image is about. Alternatively, you can  name them sequentially using the same hyphenated lesson slug
-(or an abbreviated version if the slug is rather long), followed by
-numbers to indicate which figure it is. (For example,
-`counting-frequencies-1.png`, `counting-frequencies-2.png`, and so on.)
-
-#### Use standard formats and sizes
-Make sure the images are in web-friendly formats such as PNG or JPEG and sized appropriately (both in terms of pixels and bytes).
-
-#### Including images
-Wherever you want to insert an image, use the following line of code in the body of your lesson:
-
-{% raw %}
-``` markdown
-{% include figure.html filename="IMAGE-FILENAME" caption="Caption to image" %}
 ```
-{% endraw %}
-
-You'll need to modify `IMAGE-FILENAME` and `Caption to image` according to your lesson and image. Note that you may use Markdown within caption text, for instance to mark text as bold or italic.
-
-When the Markdown is rendered by our system, this line will automatically produce HTML that looks like this:
-
-``` html
-<figure>
-    <a href="/images/LESSON-SLUG/IMAGE-FILENAME">
-       <img src="/images/LESSON-SLUG/IMAGE-FILENAME" alt="Caption to image">
-    </a>
-<figcaption>
-    Caption to image
-</figcaption>
-</figure>
+\{% include figure.html filename="IMAGE-FILENAME" caption="YOUR CAPTION" %}
```

Comment:
If you look at the original guidelines you edited, you'll see that it used the `{% raw %}` tags to show the literal liquid tag ([liquid documentation](https://shopify.github.io/liquid/tags/raw/))

Reply:
thanks